tests/Cache/FileHandlerTest.php

```diff
@@ -138,7 +138,10 @@ public function testWriteWritesCache()
         $handler->write($cache);
 
         $this->assertFileExists($file);
-        $this->assertSame($cache->toJson(), file_get_contents($file));
+
+        $actualCacheJson = file_get_contents($file);
+
+        $this->assertSame($cache->toJson(), $actualCacheJson);
```

Comment:
can't we use [assertStringEqualsFile](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertJsonFileEqualsJsonFile) or [assertStringEqualsFile](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertStringEqualsFile)